web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -1271,6 +1271,35 @@ export const formatters: Formatters = {
     desc: 'Login Rule Deleted',
     format: ({ user, name }) => `User [${user}] deleted a login rule [${name}]`,
   },
+  [eventCodes.SAML_IDP_AUTH_ATTEMPT]: {
+    type: 'saml.idp.auth',
+    desc: 'SAML IdP authentication',
+    format: ({ user, success }) => {
+      const desc =
+        success === false
+          ? 'failed to authenticate'
+          : 'successfully authenticated';
+      return `User [${user}] ${desc} with the SAML IdP`;
+    },
+  },
+  [eventCodes.SAML_IDP_SERVICE_PROVIDER_CREATE]: {
+    type: 'saml.idp.service.provider.create',
+    desc: 'SAML IdP service provider create',
+    format: ({ updated_by, name, service_provider_entity_id }) =>
+      `User [${updated_by}] added service provider [${name}] with entity ID [${service_provider_entity_id}]`,
```

Comment:
Is the entity ID important enough to be displayed on the main table, or should we just leave this info in the JSON?

Reply:
IMO yes, it essentially says what service attempted to use the IdP, which I think is worth broadcasting up front. I'm 100% fixed on this, though. If you feel strongly it can be removed.